lib/curl_addrinfo.c

```diff
@@ -50,7 +50,7 @@
 #  define in_addr_t unsigned long
 #endif
 
-#if defined(WIN32) && defined(USE_UNIX_SOCKETS)
+#if defined(USE_UNIX_SOCKETS) && !defined(WIN32)
 #include <afunix.h>
```

Comment:
This header is not generically available (on non-Windows) so this causes build failures all over. I suspect this is actually a header that's introduced in some recent Windows sdk that enables unix domain sockets?

Reply:
Oh, indeed, I’ll submit an update in a few.